packages/vscode-extension/src/debugging/DebugAdapter.ts

```diff
@@ -204,21 +286,73 @@ export class DebugAdapter extends DebugSession {
     );
   }
 
-  private createVariableForOutputEvent(args: CDPRemoteObject[]) {
-    // we create empty object that is needed for DAP OutputEvent to display
-    // collapsed args properly, the object references the array of args array
-    const argsObjectDapID = this.variableStore.pushReplVariable(
-      args.map((arg: CDPRemoteObject, index: number) => {
+  private async createVariableForOutputEvent(args: CDPRemoteObject[]) {
+    const prepareVariables = await Promise.all(
+      args.map(async (arg: CDPRemoteObject, index: number) => {
+        // if (arg.type === "object") {
```

Comment:
The first approach is to create an object split into chunks - the downside is that we need to get properties recursively